browser/themes/theme_properties.cc

```diff
@@ -0,0 +1,89 @@
+    // Window when the window is innactive, tabs and also tab bar everywhere except active tab
+    case ThemeProperties::COLOR_FRAME_INACTIVE:
+    case ThemeProperties::COLOR_BACKGROUND_TAB_INACTIVE:
+      return incognito ? SkColorSetRGB(0x71, 0x75, 0x79) : SkColorSetRGB(0xC8, 0xCE, 0xC8);
+    // Active tab and also the URL toolbar
+    // Parts of this color show up as you hover over innactive tabs too
+    case ThemeProperties::COLOR_TOOLBAR:
+    case ThemeProperties::COLOR_DETACHED_BOOKMARK_BAR_BACKGROUND:
+    case ThemeProperties::COLOR_CONTROL_BACKGROUND:
+    case ThemeProperties::COLOR_TOOLBAR_BOTTOM_SEPARATOR:
+      return incognito ? SkColorSetRGB(0x91, 0x95, 0x99) : SkColorSetRGB(0xF6, 0xF7, 0xF9);
+    case ThemeProperties::COLOR_TAB_TEXT:
+      return SkColorSetRGB(0x22, 0x23, 0x26);
+    case ThemeProperties::COLOR_BOOKMARK_TEXT:
+    case ThemeProperties::COLOR_BACKGROUND_TAB_TEXT:
+      return SkColorSetRGB(0x22, 0x23, 0x26);
+    default:
+      return base::nullopt;
+  }
+}
+#endif
+
+base::Optional<SkColor> MaybeGetDefaultColorForBraveUiDevChannel(int id, bool incognito) {
+  switch (id) {
+    // Applies when the window is active, tabs and also tab bar everywhere except active tab
+    case ThemeProperties::COLOR_FRAME:
+    case ThemeProperties::COLOR_BACKGROUND_TAB:
+      return incognito ? SkColorSetRGB(0x68, 0x6B, 0x6E) : SkColorSetRGB(0x58, 0x5B, 0x5E);
+    // Window when the window is innactive, tabs and also tab bar everywhere except active tab
+    case ThemeProperties::COLOR_FRAME_INACTIVE:
+    case ThemeProperties::COLOR_BACKGROUND_TAB_INACTIVE:
+      return incognito ? SkColorSetRGB(0x58, 0x5B, 0x5E) : SkColorSetRGB(0x48, 0x4B, 0x4E);
+    // Active tab and also the URL toolbar
+    // Parts of this color show up as you hover over innactive tabs too
+    case ThemeProperties::COLOR_TOOLBAR:
+    case ThemeProperties::COLOR_DETACHED_BOOKMARK_BAR_BACKGROUND:
+    case ThemeProperties::COLOR_CONTROL_BACKGROUND:
+    case ThemeProperties::COLOR_TOOLBAR_BOTTOM_SEPARATOR:
+      return incognito ? SkColorSetRGB(0x32, 0x33, 0x36) : SkColorSetRGB(0x22, 0x23, 0x26);
+    case ThemeProperties::COLOR_TAB_TEXT:
+      return SkColorSetRGB(0xF7, 0xF8, 0xF9);
+    case ThemeProperties::COLOR_BOOKMARK_TEXT:
+    case ThemeProperties::COLOR_BACKGROUND_TAB_TEXT:
+      return SkColorSetRGB(0x81, 0x85, 0x89);
+    default:
+      return base::nullopt;
+  }
+}
+
+}  // namespace
+
+// Returns a |nullopt| if the UI color is handled by Brave
+base::Optional<SkColor> MaybeGetDefaultColorForBraveUi(int id, bool incognito) {
+#if !defined(OFFICIAL_BUILD)
+  return MaybeGetDefaultColorForBraveUiDevChannel(id, incognito);
+#else
+  switch (chrome::GetChannel()) {
+    case version_info::Channel::STABLE:
+      return MaybeGetDefaultColorForBraveUiReleaseChannel(id, incognito);
+    case version_info::Channel::DEV:
+    case version_info::Channel::BETA:
```

Comment:
@bbondy In the description you mention that you want Beta builds to get the stable theme by default - should this case be above?

Reply:
Ya I prefer to keep beta as close to release as possible, good catch. Will update.